### PR TITLE
update bert-ofa demo

### PR DESCRIPTION
**Pull request by ceci3**

### CODE REVIEW

File: demo/ofa/bert/run_glue_ofa.py [hunk 1]
```diff
@@ -23,25 +23,27 @@
 import paddle
 import paddle.nn.functional as F
 from paddle.io import DataLoader
-from paddle.metric import Accuracy
+from paddle.metric import Metric, Accuracy, Precision, Recall
```
Comment (LiuChiachi) [inline demo/ofa/bert/run_glue_ofa.py]: 这个`Metric`好像没有用到，PaddleNLP里我好像也多import了，我也一起改一下吧:)

File: docs/zh_cn/nlp/paddlenlp_slim_ofa_tutorial.md [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 ## 基于PaddleNLP repo代码进行压缩
 本教程基于PaddleSlim2.0及之后版本、Paddle2.0rc1及之后版本和PaddleNLP2.0beta及之后版本，请确认已正确安装Paddle、PaddleSlim和PaddleNLP。
```
Comment (LiuChiachi) [inline docs/zh_cn/nlp/paddlenlp_slim_ofa_tutorial.md:21]: 文档这里目前可以更新成PaddleNLP2.0rc，rc版本号一直在更新，5.20应该要发2.0.0正式版，目前安装命令是
```
pip install --upgrade paddlenlp>=2.0.0rc -i https://pypi.org/simple
```
可以安装最新的rc版本

Comment (ceci3) [reply docs/zh_cn/nlp/paddlenlp_slim_ofa_tutorial.md:21]: done,thanks

File: demo/ofa/bert/run_glue_ofa.py (hunk 1 above)
Comment (ceci3) [reply demo/ofa/bert/run_glue_ofa.py]: done, thanks
